### PR TITLE
Adding CUDA licensing note.

### PR DESCRIPTION
**Pull request by grainnejenningsRH**
Adds a CUDA licensing note to where we reference the CUDA - RStudio image.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: modules/configuring-your-ide.adoc [hunk 1]
```diff
@@ -70,7 +70,9 @@ ifdef::cloud-service[]
 [IMPORTANT] 
 ====
 *Disclaimer:* 
-{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench.
+{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench. +
+
```
Comment (MelissaFlinn) [inline modules/configuring-your-ide.adoc:74]: extra plus at the end of line 73?
Can you fix the passive voice?
 ", which should be reviewed prior to use of this sample workbench."
instead, something like ". You should review their licensing terms before you use this sample workbench."

File: modules/notebook-images-for-data-scientists.adoc [hunk 2]
```diff
@@ -107,7 +107,9 @@ To use the *CUDA - RStudio Server* notebook image, you must first build it by cr
 [IMPORTANT] 
 ====
 *Disclaimer:* +
-{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench.
+{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench. +
+
+The *CUDA - RStudio Server* notebook image contains NVIDIA CUDA technology. CUDA licensing information is available at link:https://docs.nvidia.com/cuda/[https://docs.nvidia.com/cuda/] and should be reviewed prior to using this sample workbench.
 ====
```
Comment (MelissaFlinn) [inline modules/notebook-images-for-data-scientists.adoc:113]: same comments as for previous file

File: modules/options-for-notebook-server-environments.adoc [hunk 3]
```diff
@@ -351,7 +351,9 @@ a| * Python 3.9
 [IMPORTANT] 
 ====
 *Disclaimer:* +
-{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench.
+{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench. +
+
+The *CUDA - RStudio Server* notebook image contains NVIDIA CUDA technology. CUDA licensing information is available at link:https://docs.nvidia.com/cuda/[https://docs.nvidia.com/cuda/] and should be reviewed prior to using this sample workbench.
 ====
```
Comment (MelissaFlinn) [inline modules/options-for-notebook-server-environments.adoc:357]: same comments as for previous file

File: modules/configuring-your-ide.adoc (hunk 1 above)
Comment (grainnejenningsRH) [reply modules/configuring-your-ide.adoc:74]: Thanks, I've updated.

File: modules/options-for-notebook-server-environments.adoc [hunk 4]
```diff
@@ -328,7 +328,7 @@ a| * Python 3.9
 [IMPORTANT] 
 ====
 *Disclaimer:* +
-{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms, which should be reviewed prior to use of this sample workbench.
+{org-name} supports managing workbenches in {productname-short}. However, {org-name} does not provide support for the RStudio software. RStudio Server is available through link:rstudio.org[rstudio.org] and is subject to their licensing terms. You should review their licensing terms before you use this sample workbench.
```
Comment (chtyler) [inline modules/options-for-notebook-server-environments.adoc:331]: Not top priority right now, but the only thing I would suggest is removing the word "should".

**Review by chtyler — APPROVED** [review]

**Pull request merged**